v3/src/components/tool-shelf/options-button.tsx

```diff
@@ -28,7 +29,8 @@ export const OptionsShelfButton = () => {
       isWebViewModel(tile?.content) && tile?.content.setUrl(url)
     }, {
       undoStringKey: "V3.Undo.webView.show",
-      redoStringKey: "V3.Redo.webView.show"
+      redoStringKey: "V3.Redo.webView.show",
+      log: logStringifiedObjectMessage("Show web view: %@", {url})
```

Comment:
Could use `logMessageWithReplacement` to just render the url without the braces.